### ISSUE TITLE
Section 1 basic changes

### ISSUE THREAD

**Issue by lachellel** [uspki/policies#9]
referencing #6 and comments; moving here: 

- Staring in Section 1.1 - remove text starting at "Notice to Readers" to the end of the paragraph - this is boiler plate from the CA/B Forum and not applicable for an operational CP.
- Remove Section 1.2.1 - which is the CA/B forum Revision history.
- Remove Section 1.2.2 Relevant dates - also only applicable to CA/B forum version. CP should state the requirements.
- Consider pulling the appropriate sections for the Fed Bridge CP (or Common) to replace all of Section 1.3., 1.4 and 1.5.

@LarryFrank

**Comment by weirdscience** [comment]
Should the title be US Federal PKI? Why is it U.S. PKI?

**Comment by LarryFrank** [comment]
I would recommend “Federal” should be in the name somewhere.  It is intended to be restricted to the Fed (not planning to issue to non-Federal entities).  But do not have any strong preference as long as it can be distinguished from the “Federal PKI” as the term is currently being used.

Larry

**Comment by LarryFrank** [comment]
Section 1.2  Should we also add the EV OID (2.23.140.1.1)?  Will impact later sections.  Better to make that decision early to inform work on the rest of the document.

**Comment by weirdscience** [comment]
@LarryFrank Interesting, Federal PKI is refering to PIV or people and US PKI is TLS? It's not all Federal PKI?

**Comment by LarryFrank** [comment]
It is all semantics…

Just thinking that “US PKI” sounds much broader than what I think is the intended scope which is just the Fed.  “Fed Public Trust PKI”?

**Comment by lachellel** [comment]
IMO semantics matter to our community

A very common question is "What is Federal PKI? And how do I use it?"

Keeping the **Public Trust** is key to the messaging; as we have non-public trust use cases for devices in FPKI.

**Comment by lachellel** [comment]
Here's a question? 

Federal is _generally_ "executive branch agencies".   What about legislative and judicial branches?   :+1:

**Comment by LarryFrank** [comment]
In describing what the intended use is, I would not exclude them but, since they typically don’t play with the executive on things like this, not too worried about it.  They are part of the Federal government.  Do not need to say it is exclusive to the Executive.

I did a quick look – House and Senate don’t have HTTPS sites (at least not that I found).  USSC does – uses a Geo Trust cert.

The sticky part might actually be money.  Outside my expertise!

**Comment by lachellel** [comment]
- https://pulse.cio.gov/https/domains/#q=The%20Legislative%20Branch%20(Congress)
- House has PIV-I
- Senate has PIV-I
- US Courts have PIV-I
- U.S. Government Publishing Office - is a shared service provider - and Legislative branch

We work together!  I'd like to ensure everything we propose is inclusive.

**Comment by konklone** [comment]
The legislative and judicial branches do use plenty of HTTPS (and increasingly so since M-15-13, even though they're not covered). The House in particular has done a lot of transition work, and the Senate is in-process (at a more senatorial pace).

In terms of the name, "US PKI" is a placeholder, but it's there because I think there's a desire to not have this root confused, in operations or technical standards, with how the web community knows the existing "Federal PKI". Their architectures will be very different, and so distinct naming is important. 

And hey, perhaps state and local will be interested in using certificates from this CA for their .gov sites. At least on the GSA side, I would like to see us offer free DV certificates to any `.gov` server which wants one. I think that would be the most powerful way to avoid other civilian agencies standing up their own CA, and the most effective way at getting HTTPS deployment to the level of coverage and resilience it needs to be for `.gov`.

**Comment by jaspenc** [comment]
This may be a bit orthogonal to the discussion at hand, but about 3 years ago NIST and NSA worked together to develop a Reference CP primarily for the "Publicly-trusted PKI" - which is essentially what we're developing here.  It was eventually released as NISTIR 7924.  Has this been consulted at all?  Has the decision been made that it is not germane?  Since it's a NIST document, I just want to be sure we've done 'due diligence'.

**Comment by lachellel** [comment]
@jaspenc 

I think it's an _input_ - the core focus is to ensure alignment with **CA/B Forum** Baseline Requirements.  

The NISTIR can help inform some areas.

**Comment by LarryFrank** [comment]
@jaspenc 

I think Common CP would be a better starting point for input than the NISTR - which is (I believe) based on a lower security standard that required for the Fed PKI.  Also, to be complete, the CA/B forum network security standard needs to be mapped to standard Fed network controls, etc to see if there is any delta that needs to be addressed.

**Comment by jaspenc** [comment]
OK.  I just wanted to be sure the NISTIR was on the radar screen at least as a resource document.  NIST/NSA started with COMMON when they wrote it and they did some interesting work in Section 6.  I agree that we need to be sure there is alignment with Federal security controls, which again is Section 6.  As for lower security standard - they tried to embrace a range, I believe.

**Comment by jaspenc** [comment]
Section 1.1 - first paragraph - Change 'an integrated set of technologies. . .' to 'the integrated set of technologies. . .' otherwise, you leave the impression there might be an alternative set.
Section 1.1 - Notice to Readers - 
1st paragraph indicates this CP describes 'a subset of the requirements. . .' I suggest the CP describes 'the set of requirements.'  The CP is the governing document for the US Federal Device Root CA and as such must include all the requirements for issuing US Federal (I'd lose the word 'issued' here) Publicly-Trusted Certificates.  
2nd paragraph - first sentence.  This is related to the comment above - I hope these requirements address ALL of the issues otherwise you leave people guessing as to what the unaddressed requirements are, or you have to reference another document where those requirements are listed - and that's not a good idea.  This CP needs to be self-contained.    
3rd paragraph - I'd reword this somewhat, talk in terms of the document "This CP addresses certificates. . ."
4th paragraph - 1st Sentence - make it 'the chain of trust' even though there will be multiple paths, it is a single chain of trust.
4th paragraph - 2nd Sentence - It is very poetic, but a bit confusing.  Recommend rewording this as follows: 'This CP must be adopted by all Certification Authorities that subordinate to the Federal Device Root CA.

**Comment by jaspenc** [comment]
Section 1.2 - I understand why we'd want to include the CAB Forum OIDs, does FPKI want to include its own OIDs so these certificates can be recognized as Federally issued via the policy OID?  Or do you intend to rely on the DN for that?

**Comment by LarryFrank** [comment]
> Section 1.2 - I understand why we'd want to include the CAB Forum OIDs, does FPKI want to include its own OIDs so these certificates can be recognized as Federally issued via the policy OID? Or do you intend to rely on the DN for that?

Being discussed under question #27 - believe the end of the discussion was there is a need for at least one Fed Policy to conform with CA/B forum Baseline requirements:  

"•A Certificate issued to a Subscriber MUST contain one or more policy identifier(s), defined by the Issuing CA, in the Certificate's certificatePolicies extension that indicates adherence to and compliance with these Requirements. CAs complying with these Requirements MAY also assert one of the reserved policy OIDs in such Certificates."
